### PR TITLE
fix: rename in app listener keys

### PR DESCRIPTION
**Pull request by Shahroz16**
closes: https://github.com/customerio/issues/issues/9190

Complete each step to get your pull request merged in. [Learn more about the workflow this project uses](https://github.com/customerio/customerio-android/develop/docs/dev-notes/GIT-WORKFLOW.md).
- [ ] [Assign members of your team](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to review the pull request.
- [ ] Wait for pull request status checks to complete. If there are problems, fix them until you see that [all status checks are passing](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fsymfony.com%2Fdoc%2F4.3%2F_images%2Fdocs-pull-request-symfonycloud.png&f=1&nofb=1).
- [ ] Wait until the pull request has been reviewed *and approved* by a teammate
- [ ] After pull request is approved, and you determine it's ready **add the label "Ready to merge"** to the pull request and it will automatically be merged.

### CODE REVIEW

File: messaginginapp/src/main/java/io/customer/messaginginapp/type/InAppEventListener.kt [hunk 1]
```diff
@@ -4,5 +4,5 @@ interface InAppEventListener {
     fun messageShown(message: InAppMessage)
     fun messageDismissed(message: InAppMessage)
     fun errorWithMessage(message: InAppMessage)
-    fun messageActionTaken(message: InAppMessage, action: String, name: String)
+    fun messageActionTaken(message: InAppMessage, action_value: String, action_name: String)
```
Comment (mrehan27) [inline messaginginapp/src/main/java/io/customer/messaginginapp/type/InAppEventListener.kt:7]: Just a question, can't we use camel case here? e.g. `actionValue` and `actionName`? 🤓

**Review by mrehan27 — APPROVED** [review]
Just one small question, rest looks good to me 👍🏻

File: messaginginapp/src/main/java/io/customer/messaginginapp/type/InAppEventListener.kt (hunk 1 above)
Comment (Shahroz16) [reply messaginginapp/src/main/java/io/customer/messaginginapp/type/InAppEventListener.kt]: Good call out, done

**Comment by levibostian** [comment]
Looks great. Is there a docs update PR already made for this change?

**Comment by Shahroz16** [comment]
Great point, added in the [docs ticket](https://github.com/customerio/docs/pull/791)

**Pull request merged**